DirectEmployers/MyJobs
# [PD-3291] manage users removing help tutorials

## What's this PR do?
Hides the help tutorials button on the side facet inside of Manage Users

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of Manage Users

## How should this be manually tested?
Pull down repository. Go into Manage Users and you should see the Help & Tutorials side facet button gone

## Any background context you want to provide?
